mint.json

```diff
@@ -41,13 +41,14 @@
   ],
   "navigation": [
     {
-      "group": "Welcome",
+      "group": "Introduction",
```

Comment:
I should double-check this doesn't break anything before merging.

Reply:
As far as I can tell it only affects the naming in the sidebar, it doesn't affect any links.